src/components/EmailNotifications/actions.js

```diff
@@ -45,8 +44,6 @@ export const actions = {
     requiresEmail: false,
   },
   [OPTOUT_ACTION]: {
-    request: switchNotificationsStatus,
-    params: [true], // disabled = true
     requiresEmail: false,
```

Comment:
Double checking here, what happens when the user presses "no, thanks"? Don't we still need to send an API request?

Reply:
Oh, I see; what we're doing is just verifying their address and not setting anything else?